### ISSUE TITLE
Replace fromNow-util with Intl.RelativeTimeFormat

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#127793]
We have [`fromNow`](https://github.com/microsoft/vscode/blob/7f4d4525011a5e75b2c725e0c016b57bf96413c2/src/vs/base/common/date.ts#L15-L15) which is obsolete because we use `Intl.RelativeTimeFormat`, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/RelativeTimeFormat#basic_format_usage

**Comment by eamodio** [comment]
@jrieken Are you suggesting that [`fromNow`](https://github.com/microsoft/vscode/blob/7f4d4525011a5e75b2c725e0c016b57bf96413c2/src/vs/base/common/date.ts#L15-L15), would internally use the new APIs? Because looking at the [`format`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/format) API there doesn't look like there is a way to format a date automatically based on the distance -- as you have to supply a particular unit.

**Comment by jrieken** [comment]
Yes, I'd say it is simple and much less code to pick a unit and then use the built-in API

**Comment by eamodio** [comment]
Well the built-in API doesn't support the shortened strings we use in the Timeline view -- so we could use the built-ins for other usages just not the Timeline view column.

**Comment by jrieken** [comment]
> Well the built-in API doesn't support the shortened strings we use in the Timeline view

Can you explain that further?

**Comment by eamodio** [comment]
We use abbreviated strings in the Timeline view -- hr = hour, mo = month, wk = week, etc, but AFAICT there is no abbreviated format with the built-ins.

**Comment by jrieken** [comment]
Like so? (running `short` and/or `narrow`)

<img width="201" alt="Screen Shot 2021-07-20 at 09 40 18" src="https://user-images.githubusercontent.com/1794099/126281425-960c0512-c32e-421c-b668-3884dba39afe.png">


```
const rtf = new Intl.RelativeTimeFormat("en", {
    localeMatcher: "best fit", // other values: "lookup"
    numeric: "always", // other values: "auto"
    style: "long", // other values: "short" or "narrow"
});
```

**Comment by eamodio** [comment]
Kind of, but we don't add the `.` nor the `ago`

![image](https://user-images.githubusercontent.com/641685/126352533-202d019c-9aff-4459-87c4-7bb2937267b3.png)
